about/faq.rst

```diff
@@ -44,15 +44,15 @@ Which platforms are supported by Godot?
 * Windows
 * macOS
 * Linux, \*BSD
-* Android (experimental)
+* Android (experimental, including Meta Quest 3)
 * `Web <https://editor.godotengine.org/>`__ (experimental)
 
 **For exporting your games:**
 
 * Windows
 * macOS
 * Linux, \*BSD
-* Android
+* Android (including Meta Quest 3)
```

Comment:
Yeah, I think it's somewhat reasonable to single out Meta Quest devices for the *XR editor*, since that seems to be the [scope of current support](https://docs.godotengine.org/en/latest/tutorials/editor/using_the_xr_editor.html#xr-devices-support). But for exported projects we shouldn't be this specific.